### PR TITLE
Added: homepage

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
please review my PR for HOMEPAGE

image
image
image
:
: 
image

check live at :- https://admiring-bassi-a13c59.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
+:root {
+    --backcolor: #222;
+    --primary: white;
+    --border-color: grey;
+    --footer-back: black;
+    --poster-border: rgb(114, 113, 113);
+    --danger: #e50914;
+}
+
```
Comment (someshtiwarii) [inline style.css:9]: delete the empty spaces

File: style.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+body {
+    padding: 0;
+    margin: 0;
+    background-color: var(--backcolor);
+    color: var(--primary);
+}
+
+.trivia-header-div {
+    background-image: url('https://bit.ly/3p1IZrt');
+    height: 600px;
```
Comment (someshtiwarii) [inline style.css:21]: try using rem

File: style.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.question-div {
+    display: flex;
+    flex-flow: column wrap;
+    width: 700px;
+    justify-content: center;
+    margin: auto;
+    gap: 20px;
+    margin-top: 100px;
```
Comment (someshtiwarii) [inline style.css:129]: can use rem here

**Review by someshtiwarii — APPROVED** [review]
try using rem instead of px , rest looks good !

**Review by manojkumar8400 — APPROVED** [review]
Looks good to me go to.

File: style.css (hunk 2 above)
Comment (Amandeep-sinha) [reply style.css:21]: okay

**Pull request merged**